src/som/interpreter/nodes/SOMNode.java

```diff
@@ -33,9 +33,11 @@
 
 @TypeSystemReference(Types.class)
 public abstract class SOMNode extends Node implements MateNode {
+  protected final SourceSection sourceSection;
 
   public SOMNode(final SourceSection sourceSection) {
-    super(sourceSection);
+    super();
```

Comment:
This change looks strange. What's going on here? (there were other changes to super calls, too)